pkg/cloudprovider/BUILD

```diff
@@ -20,9 +20,6 @@ filegroup(
 
```

Comment:
Whats left in cloudprovider? It seems like the whole pkg/cloudprovider directory should just be deleted.